Automattic/wp-calypso
# Add Go-Mobile-On-Mobile as a Primary Task on mobile

#### Changes proposed in this Pull Request

* Add a primary Task only displayed on mobile.

#### Testing instructions



* Start your browser in "device-mode"
* Go to http://calypso.localhost:3000/home/
* depending on the device you are "emulating", you should see:
![img](https://user-images.githubusercontent.com/1270189/81215975-04b09c80-8f8f-11ea-92ba-355faabd57cd.png)



Fixes #41880
